### DISCUSSION TITLE
Error trying to import brushes. Also feature suggestion: import brushes from Procreate.

### DISCUSSION THREAD

**Discussion by Mirakelpung** [keiwando/toonsquid#398] [General]
Hi. I recently downloaded the Aaron Blaise Custom Photoshop Brushes - Set 1 from creatureartteacher.com and got 2 .ABR files.
Both of these files were possible to open on Procreate, however when I try to open these in Toonsquid one of them returns me with an error message saying the file is either not supported or it is corrupt.

However both the files are .ABR files which the program says it supports, both of these files worked on procreate, and one of them worked on Toonsquid. I also paid money for these brushes so they're professional brushes, and I downloaded them twice in case something had happened to the old files. Got any idea what is going on here?

Either way, I would love for the ability to import brushes and brushpacks from Procreate since I use both Procreate and Toonsquid in tandem.

**Comment by VeiloPot** [comment]
> Hi. I recently downloaded the Aaron Blaise Custom Photoshop Brushes - Set 1 from creatureartteacher.com and got 2 .ABR files.
> 
> Both of these files were possible to open on Procreate, however when I try to open these in Toonsquid one of them returns me with an error message saying the file is either not supported or it is corrupt.
> 
> 
> 
> However both the files are .ABR files which the program says it supports, both of these files worked on procreate, and one of them worked on Toonsquid. I also paid money for these brushes so they're professional brushes, and I downloaded them twice in case something had happened to the old files. Got any idea what is going on here?
> 
> 
> 
> Either way, I would love for the ability to import brushes and brushpacks from Procreate since I use both Procreate and Toonsquid in tandem.

Keiwan mentioned importing procreate brushes couldnt work for now.

**Comment by Mirakelpung** [comment]
Ah, fair enough. I still don't understand why one of the ABR files won't work though.

**Reply by ota-works** [reply]
Yeah, the ABR brush import are a bust currently. I wanted to import unique inking brushes so I could clean up my animations with them, but na-dah. They were horrible and lacked the uniqueness I was looking for.
Maybe that would be fixed in future updates.

**Comment by keiwando** [comment]
At least one of the brushes in that file uses a 16 bit texture, which is currently not supported by ToonSquid's ABR import. I've noted this down to add support for this in the future. There is nothing wrong with the file itself.
